css/main.css

```diff
@@ -6,3 +6,176 @@ Green: #00B172
 White: #FFF
 
 */
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Roboto', sans-serif;
+}
+
+body {
+    max-width: 1480px;
+    margin: 0 auto;
+}
+
+.nav-logo {
+    height: 60px;
+    float: left;
+
+}
+
+.nav-bar {
+    height: 60px;
+    padding: 20px;
+}
+
+.nav-bar ul {
+    float: right;
+}
+
+.nav-bar ul li {
+    display: inline-block;
+}
+
+.nav-bar ul li a {
+    text-decoration: none;
+    color: #1A1A1A;
+    padding: 20px;
+    font-size: 20px;
+    line-height: 60px;
+}
+
+.nav-bar ul li a:hover {
+    background-color: rgb(238, 225, 225);
+}
+
+.photo-stage {
+    background-image: url("../images/landing.jpg");
+    height: 85vh;
```

Comment:
You shouldn't be using vh here, because it depends or the size of the window, id you reduce it, the section end up disappearing, here you can use px